packages/desktop-client/src/components/BankSyncStatus.tsx

```diff
@@ -58,8 +59,10 @@ export function BankSyncStatus() {
                   iconStyle={{ color: theme.pillTextSelected }}
                 />
                 <Text style={{ marginLeft: 5 }}>
-                  Syncing... {accountsSyncingCount} account
-                  {accountsSyncingCount > 1 && 's'} remaining
+                  <Trans>
```

Comment:
Harder to test because I'd have to hook it up to a server but I imagine this will cause the same error too